src/frame/js/actions/conversation.js

```diff
@@ -516,7 +516,7 @@ export function handleUserConversationResponse({appUser, conversation, hasPrevio
             setUser(appUser),
             setConversation({
                 ...conversation,
-                hasMoreMessages: hasPrevious
+                hasMoreMessages: !!hasPrevious
```

Comment:
In some responses we don't include `messages` or `hasPrevious`, which was leading to react throwing an error "this prop is required but undefined was passed". This makes sure to coerce `undefined` to `false`